meetings/agenda.md

```diff
@@ -7,3 +7,22 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+- List organizations offering support?
+  Matthew has had a few companies and contractors ask about having their
+  information listed on the site as providing commercial support, and a few
+  others asking about what commercial support offerings exist.
+
+  He asked the LF legal representative about if this is allowed, and, if so,
+  what stipulations there might be around it. His response:
+
+  > [Y]ou can either have a formal program with criteria or an informal program.
+  > If it's a formal program, you list people that qualify. If it's informal, you
+  > list people that ask.  If you find that people are just asking to get the logo
+  > listed without really supporting, I would recommend a charter for a support
+  > program that would list minimum criteria.
+
+  The argument for listing support providers is that it helps establish that
+  there's a healthy ecosystem.
+
+  Do we want to list support providers? If so, do we want to do so informally
```

Comment:
If I can't make the meeting, I'd favour having minimum criteria for this kind of thing, and making it "formal". Have a clear definition of what being a support means, and what it achieves. Note: that doesn't necessarily have to be monetary, and the criteria could be reasonably vague, e.g. "by agreement of the TSC, the provider benefitted the project in some significant way to warrant recognition" for example